saleor/product/models/base.py

```diff
@@ -27,13 +27,29 @@
 class CategoryManager(Manager):
 
     def prefetch_for_api(self):
+        # type: () -> django.db.models.QuerySet
+        """ Return Category QuerySet with added prefetch_related for api"""
         return self.get_queryset().prefetch_related(
             'products__images', 'products__variants',
             'products__variants__stock')
 
 
 @python_2_unicode_compatible
 class Category(MPTTModel):
+    """ Category Model, inheritance from MPTTModel
+
+    Fields:
+    name -- name of category (Char Field, max length 128)
+    slug -- slug for category, (Slug Field, max length 50)
+    description -- description, (Text Field, required False)
+    parent -- parent Category (Foreign Key, required False)
+    children -- children Categories (QuerySet)
+    hidden -- does category is hidden (Boolean Field)
+    products -- producsts in category
+
+    objects -- CategoryManager instance
```

Comment:
This crosses into "documenting how Django works" territory. Also it's not an instance field by any means.